server/server.js

```diff
@@ -38,6 +38,17 @@ function prodLangs() {
 
 const availableLanguages = conf.l10n_dev ? allLangs() : prodLangs();
 
+if (conf.env !== 'production') {
```

Comment:
Just to confirm, the send.stage.mozaws.net server runs in a NODE_ENV="production" environment, correct?